<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VHttpProxyWidget</class>
 <widget class="QWidget" name="VHttpProxyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInbound">
      <attribute name="title">
       <string>Inbound</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="glInbound">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget2">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tabTcpServer">
           <attribute name="title">
            <string>TCP Server</string>
           </attribute>
           <widget class="QWidget" name="gridLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>160</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="glTcpServer">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tabSslServer">
           <attribute name="title">
            <string>SSL Server</string>
           </attribute>
           <widget class="QWidget" name="gridLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>160</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="glSslServer">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabOutbound">
      <attribute name="title">
       <string>Outbound</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="glOutbound">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
